gatsby-config.js

```diff
@@ -106,6 +106,12 @@ module.exports = {
           type: 'expanded_bubble'
         }
       }
+    },
+    {
+      resolve: `gatsby-plugin-env-variables`,
+      options: {
+        allowList: [`BRANCH`]
```

Comment:
It cost me a little bit to understand the **HOW**, but now I understand that as we need a `env` variable that comes from other source we need this lib in order to allow our runtime process to "look" at environment variable and inject values (in this case `BRANCH`) but it is because this variable comes from other source otherwise it would work out of the box. Is that right?

Reply:
Actually you are right, we need to think how this will work in local dev. Maybe we can check for this change only when it's running on netlify?